api_tests/lib_sdk/actors/actor.ts

```diff
@@ -178,15 +178,15 @@ export class Actor {
 
     public async accept() {
         if (!this.swap) {
-            throw new Error("Cannot accept inexistent swap");
+            throw new Error("Cannot accept non-existent swap");
         }
 
         await this.swap.accept(Actor.defaultActionConfig);
     }
 
     public async fund() {
         if (!this.swap) {
-            throw new Error("Cannot fund inexistent swap");
+            throw new Error("Cannot fund non-existent swap");
```

Comment:
```suggestion
            throw new Error("Cannot fund nonexistent swap");
```

Reply:
Cool I learned about hyphens, the web seems to think if it is a 'real' word (which nonexistent is) it does _not_ get hyphens and if it is made up (which I also learned is totally acceptable) it gets hyphens :)